code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -8,9 +8,19 @@
 	resistance_flags = FIRE_PROOF
 	var/mob/affecting = null
 	var/deity_name = "Christ"
+	/// Is the sprite of this bible customisable
+	var/customisable = FALSE
+
+	// ALL OF THESE MUST BE THE SAME LENGTH
+	/// List of bible variant names, used for the radial menu
+	var/static/list/bible_names = list("Bible", "Koran", "Scrapbook", "Creeper", "White Bible", "Holy Light", "PlainRed", "Tome", "The King in Yellow", "Ithaqua", "Scientology", "the bible melts", "Necronomicon", "Greentext")
```

Comment:
Maybe `ASSERT()` that these 3 are the same length in `Initialize()`

Reply:
Or maybe make it a list of (associative) lists.
```dm
var/static/variants = list(
  list("name" = "Bible", "state" = "bible", "inhand" = "bible"),
  list("name" = "Koran", "state" = "koran", "inhand" = "koran"),
  ...
)
```